### ISSUE TITLE
v4.0.1 POSIX syntax error: `/opt/start-runner-service.sh: 4: [[: not found`

### ISSUE THREAD

**Issue by strophy** [philips-labs/terraform-aws-github-runner#3396]
Using v4.0.1 to build a custom AMI and attempting to use it with the JIT config results in the following error from the runner on startup:

```
Starting the runner as user ubuntu
Starting the runner in ephemeral mode
/opt/start-runner-service.sh: 4: [[: not found
Starting without JIT config
An error occurred: Not configured. Run config.(sh/cmd) to configure the runner.
Runner listener exit with terminated error, stop the service, no retry needed.
Exiting runner...
Runner has finished
Stopping cloudwatch service
Terminating instance
{
    "TerminatingInstances": [
        {
            "CurrentState": {
                "Code": 32,
                "Name": "shutting-down"
            },
            "InstanceId": "i-08b5a096fc4f265cd",
            "PreviousState": {
                "Code": 16,
                "Name": "running"
            }
        }
    ]
}
```

I believe the issue is the lack of a shebang and calling the script with `nohup /opt/start-runner-service.sh` which probably defaults to the `sh` shell. Modifying the condition to using POSIX-style single brackets and comparison operators like `if [ "$enable_jit_config" = "true" ]; then` resolves the issue. I did a bit of testing, but wasn't able to get `nohup` to run the script using bash by adding a shebang, maybe `nohup bash /opt/start-runner-service.sh` would also fix it?

**Comment by strophy** [comment]
One more issue with this new config item: in the v4.0.1 release notes, it recommends reading https://github.com/philips-labs/terraform-aws-github-runner/pull/3393 for details on how to handle configuration. This PR suggests the config variable name is `enable_jit_configuration` but in fact it is `enable_jit_config`, resulting in confusion.

**Comment by npalm** [comment]
This issue is solved in release 4.0.2 in PR #3407 thanks to @mcieno
